packages/components/src/internal/components/assay/actions.ts

```diff
@@ -243,6 +244,32 @@ export function deleteAssayRuns(
     });
 }
 
+export function getImportItemsForAssayDefinitionsQM(
```

Comment:
Any reason to try and consolidate these two `getImportItemsForAssayDefinitions` functions? It looks like they only use a couple of things from the sampleModel.

Reply:
Because getImportItemsForAssayDefinitions is used outside of ui-components/AssayImportSubMenuItem, and I was trying to make the transition a bit easier, plus in the long run we really only need one implementation, so when we drop QGM we can just remove the old version. I could replace the current version with one that takes queryInfo, filters, and selectionKey args to generalize it though.